guides/common/modules/ref_global-parameters-for-registration.adoc

```diff
@@ -0,0 +1,24 @@
+[id="global-parameters-for-registration_{context}"]
+= Global Parameters for Registration
+
+You can configure the following global parameters by navigating to *Configure* > *Global Parameters*:
+
+* The `host_registration_remote_execution` parameter is used in the `remote_execution_ssh_keys` snippet.
+If it is set to `true`, the registration enables remote execution on the host.
+The default value is `true`.
+* The `host_registration_insights` parameter is used in the `insights` snippet.
+ifdef::satellite[]
+The default value is `true`.
+endif::[]
+ifndef::satellite[]
+The default value is `false`.
+endif::[]
+Set the parameter type to `boolean` to override the parameter value.
+If the parameter is set to `false`, it prevents {Project} and the Insights client from uploading Inventory reports to your {RHCloud}.
+* The `host_packages` parameter is for installing packages on the host.
+* The `remote_execution_ssh_keys`, `remote_execution_ssh_user`, `remote_execution_create_user` and `remote_execution_effective_user_method` parameters are used in the `remote_execution_ssh_keys` snippet.
+For more details, see the snippet.
+* The `encrypt_grub` parameter is to enable encryption of a bootloader password for the host.
```

Comment:
Where is this one used, I don't remember it in the registration templates

Reply:
Hmmm, in _Kickstart default_ :D
No idea who added it here. I'll remove it?